manager/integration/tests/test_recurring_job.py

```diff
@@ -1997,15 +1998,19 @@ def test_recurring_job_backup(set_random_backupstore, client, batch_v1_api):  #
     write_volume_random_data(volume1)
     write_volume_random_data(volume2)
     time.sleep(60 - WRITE_DATA_INTERVAL)
-    wait_for_backup_count(client.by_id_backupVolume(volume1_name), 1)
-    wait_for_backup_count(client.by_id_backupVolume(volume2_name), 1)
+    wait_for_backup_count(
+        find_backup_volume(client, volume1_name, retry=RETRY_BACKUP_COUNTS), 1)
+    wait_for_backup_count(
+        find_backup_volume(client, volume2_name, retry=RETRY_BACKUP_COUNTS), 1)
 
     # 2nd job
     write_volume_random_data(volume1)
     write_volume_random_data(volume2)
     time.sleep(60 - WRITE_DATA_INTERVAL)
-    wait_for_backup_count(client.by_id_backupVolume(volume1_name), 2)
-    wait_for_backup_count(client.by_id_backupVolume(volume2_name), 2)
+    wait_for_backup_count(
+        find_backup_volume(client, volume1_name, retry=RETRY_BACKUP_COUNTS), 2)
+    wait_for_backup_count(
+        find_backup_volume(client, volume2_name, retry=RETRY_BACKUP_COUNTS), 2)
```

Comment:
Do we really need such a long retry five minutes for `find_backup_volume(client, volume1_name, retry=RETRY_BACKUP_COUNTS)` with `RETRY_BACKUP_COUNTS = 300`?

Reply:
Why need retries here?

Reply:
It would return immediately if it gets the corresponding backup volume.
But Yes, we can shorten this for this 16M volume.

Reply:
>Why need retries here?

Because there is a short time that the backup volume is not created when the backup is created.

Reply:
Modified to 30 times.